e2e/stack_test.go

```diff
@@ -87,11 +107,21 @@ func TestE2E(t *testing.T) {
 	}
 	t.Log("Monomer can sync")
 
+	getTx, err := client.Tx(ctx, bftTx.Hash(), false)
+
+	require.NoError(t, err)
+	require.Equal(t, abcitypes.CodeTypeOK, getTx.TxResult.Code, "txResult.Code is not OK")
+	require.Equal(t, bftTx, getTx.Tx, "txBytes do not match")
+
+	txBlock, err := monomerClient.BlockByNumber(ctx, big.NewInt(getTx.Height))
+	require.NoError(t, err)
+	require.Len(t, txBlock.Transactions(), 2)
+
```

Comment:
Can we check that the state has been modified? i.e., that `userTxKey` is set to `userTxValue`?

Reply:
Actually, disregard. Let's leave that in the unit tests.